doc/development/deprecations.rst

```diff
@@ -55,6 +55,16 @@ Pending deprecations
 
   - Deprecated in v0.31
 
+* ``qml.qchem.jordan_wigner`` is deprecated, and usage will now raise a warning.
+  Use ``qml.jordan_wigner`` instead. List input to define the fermionic operator
+  is also deprecated; the fermionic operators ``qml.FermiA``, ``qml.FermiC``,
+  ``qml.FermiWord`` and ``qml.FermiSentence`` should be used instead. See the
+  :mod:`pennylane.fermi` module documentation and the
+  `Fermionic Operator <https://pennylane.ai/qml/demos/tutorial_fermionic_operators>`_
+  tutorial for more details.
+
+  - Deprecated in v0.32
```

Comment:
For this one, I think it's ok to commit to a removal target. @soranjh do you have a feel for how sensitive this change might be for users? Would one release be enough of warning, or would two be better? I'm on board with 0.33 for removal.

Reply:
We have a powerful replacement for this function, so I think one release will be enough.